tests/e2e/code-first.spec.ts

```diff
@@ -43,6 +43,38 @@ describe('Code-first', () => {
       });
   });
 
+  it('should return the search result', () => {
```

Comment:
Added this test just to confirm that union types were working properly in non-federated code-first approaches.